shotover-proxy/src/protocols/cassandra_codec.rs

```diff
@@ -145,14 +145,14 @@ impl CassandraCodec {
                                         Value::NULL => to_int(-1_i32),
                                         Value::Bytes(x) => x.to_vec(),
                                         Value::Strings(x) => Vec::from(x.as_bytes()),
-                                        Value::Integer(x) => {
+                                        Value::Integer(x, _) => {
                                             let mut temp: Vec<u8> = Vec::new();
                                             let _ = temp.write_i64::<BigEndian>(*x).unwrap();
```

Comment:
is this correct for integer sizes?

Reply:
Added a commit properly writing the ints